### PR TITLE
docs: expand OSS usage documentation

### PR DESCRIPTION
**Pull request by sdowell**

### CODE REVIEW

File: docs/usage.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# Using Config Sync
```
Comment (janetkuo) [inline docs/usage.md:1]: Suggest add a link to this doc in README as well

File: docs/usage.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+This document provides instructions on how to use Config Sync in a kubernetes
```
Comment (janetkuo) [inline docs/usage.md:3]: nit: capitalize Kubernetes following the K8s doc style guide https://kubernetes.io/docs/contribute/style/style-guide/#kubernetes-io-word-list

File: docs/usage.md (hunk 1 above)
Comment (janetkuo) [inline docs/usage.md:1]:
```suggestion
# Using Open Source Config Sync
```

File: docs/usage.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+cluster. This assumes Config Sync has already been [installed on the cluster](./installation.md).
+Installing Config Sync creates the Custom Resource Definitions (CRDs) which are
+used to configure Config Sync.
+
+## Config Sync CRDs
+
+Config Sync uses a concept of [Root repositories and Namespace repositories]. These
+concepts map to the RootSync and RepoSync CRDs, respectively.
+
+Installing Config Sync using Anthos Config Management includes the ConfigManagement CRD
```
Comment (janetkuo) [inline docs/usage.md:13]: Add a link to the commercial version of installation on the GCP doc site (the OSS installation guide needs a similar link if it's not already there).

File: docs/usage.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+
+This document provides instructions on how to use Config Sync in a kubernetes
+cluster. This assumes Config Sync has already been [installed on the cluster](./installation.md).
+Installing Config Sync creates the Custom Resource Definitions (CRDs) which are
+used to configure Config Sync.
+
+## Config Sync CRDs
+
+Config Sync uses a concept of [Root repositories and Namespace repositories]. These
+concepts map to the RootSync and RepoSync CRDs, respectively.
```
Comment (janetkuo) [inline docs/usage.md:11]:
```suggestion
concepts map to the `RootSync` and `RepoSync` CRDs, respectively.
```

File: docs/usage.md (hunk 3 above)
Comment (janetkuo) [inline docs/usage.md:13]:
```suggestion
Installing Config Sync using Anthos Config Management includes the `ConfigManagement` CRD
```

File: docs/usage.md [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+
+Installing Config Sync using Anthos Config Management includes the ConfigManagement CRD
+and Operator, but those are not included in OSS installation.
```
Comment (janetkuo) [inline docs/usage.md:14]:
```suggestion
and ACM Operator, but those are not included in OSS installation.
```

File: docs/usage.md [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+
+## API Fields
+
+See [RootSync/RepoSync fields] for a high level overview of the API fields exposed by the
```
Comment (janetkuo) [inline docs/usage.md:18]:
```suggestion
See [`RootSync`/`RepoSync` fields] for a high level overview of the API fields exposed by the
```

File: docs/usage.md [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+RootSync and RepoSync objects. References to the ConfigManagement CRD and operator
```
Comment (janetkuo) [inline docs/usage.md:19]:
```suggestion
`RootSync` and `RepoSync` objects. References to the `ConfigManagement` CRD and ACM Operator (i.e. the `config-management-operator` Deployment in the `config-management-system` namespace)
```

**Review by janetkuo — COMMENTED** [review]
A few suggestions and lgtm otherwise

File: docs/usage.md (hunk 1 above)
Comment (sdowell) [reply docs/usage.md]: done

Comment (sdowell) [reply docs/usage.md]: done

File: docs/usage.md (hunk 4 above)
Comment (sdowell) [reply docs/usage.md]: done

File: docs/usage.md (hunk 3 above)
Comment (sdowell) [reply docs/usage.md]: done

Comment (sdowell) [reply docs/usage.md]: done

File: docs/usage.md (hunk 5 above)
Comment (sdowell) [reply docs/usage.md]: done

File: docs/usage.md (hunk 6 above)
Comment (sdowell) [reply docs/usage.md]: done

**Review by janetkuo — APPROVED** [review]
/lgtm

**Pull request merged**